code/obj/item/rcd.dm

```diff
@@ -415,6 +418,72 @@ Broken RCD + Effects
 						W.attach_light_fixture_parts(user, LB, TRUE)
 						log_construction(user, "built a light fixture to a wall ([W])")
 
+ // Express limb surgery with an RCD
+	attack(mob/living/carbon/human/M as mob, mob/living/carbon/user as mob)
```

Comment:
```suggestion
	attack(mob/living/carbon/human/M, mob/living/carbon/user)
```
`as X` doesn't do anything outside of verbs, it just shows up in a lot of places because it would be a pain to remove

Reply:
Removed them